addons/l10n_ro_edi_stock_batch/models/l10n_ro_edi_stock_etransport_interface.py

```diff
@@ -0,0 +1,67 @@
+from odoo import api, fields, models, _
+
+
+class L10nRoEdiStockETransportInterface(models.Model):
+    _inherit = 'l10n_ro.edi.stock.etransport.interface'
+
+    batch_id = fields.Many2one(comodel_name='stock.picking.batch', default=None)
+
+    # Compute Methods
```

Comment:
```suggestion
    ################################################################################
    # Compute Methods
    ################################################################################
```
To make it more visible, and look cool ;)  (same for the "Template Helpers" and "Other").